content/posts/2023-03-05-gather-town-building.md

```diff
@@ -0,0 +1,186 @@
+#### Speakers' Room 
+
+![](images/2023-03-05-gather-town-building/19.png)
+
+After watching talk videos, attendees can walk into the speakers room according to the program schedule. In this area, speakers and attendees can communicate freely and directly. For everyone who got questions, this was the perfect time to approach the speakers.
+
+![](images/2023-03-05-gather-town-building/20.png)
+
+
+## Sponsor & Community booth 
+#### Sponsor booth
+This year, we have a lot of sponsors to support us. And also we provide booths for sponsors to interact with attendees. Our booth is an imitation of Taiwan's shopping street so that foreign attendees can also feel the intense atmosphere of Taiwan. <i> - written by Allen - </i>
+
+![](images/2023-03-05-gather-town-building/21.png)
+
+#### Community booth
+
+If the attendees want to interact with communities like KIWI PyCon, Taichung.py, MOPCON, PyCon MY, PYCON HK, PYCON JP, and PyCon PH. They can go to the beach. We designed a marketplace called "台味市集" and put lots of booths for each community. <i> - written by Pochun - </i>
+
+![](images/2023-03-05-gather-town-building/22.png)
+
+![](images/2023-03-05-gather-town-building/23.png)
+
+Also, we have a podcast show called PyCast, held by volunteers.
+We put the episode links at the corners of the space. 
+
+![](images/2023-03-05-gather-town-building/24.png)
+
+
+![](images/2023-03-05-gather-town-building/25.png)
+
+
+
+
+## Booth Game
+
+#### Taiwan-travel
+This year, we designed a Monopoly as our main space. Working through the path, you can see many icons in the squares. Each icon square has a challenge. Take the challenges to win the prize.   <i> - written by Fish - </i>
+
+![PyCon APAC 2022 Monopoly]![](images/2023-03-05-gather-town-building/26.jpg)
+> PyCon APAC 2022 Monopoly
+
+#### Mistery-space 
+There are two hidden levels of our booth game, attendees could find the secret entrance to the mystery,and clear the mission to get points. <i> - written by Vivian - </i>
+
+![](images/2023-03-05-gather-town-building/27.png)
+
+> The entrance to the mystery.
+
+![](images/2023-03-05-gather-town-building/28.png)
+
+> Get the drink: buy a papaya milk for the bear.
+
+![](images/2023-03-05-gather-town-building/29.png)
+
+> The multiverse: find the way out of the multiverse by finding the difference of three similar scenes.
+
+![](images/2023-03-05-gather-town-building/30.png)
+
+#### Discord 
+This year, we not only use discord as our main communication media, but also use it to run our booth game to increase the ineractivity. You can find hints on the gather town map and follow the instructions to play the game. The goal is to answer as many questions as you can to aquire higher points. Attendees with more points will have a higher chance to win awards and gifts. There are also hidden missions on gather town and hidden discord commands to be discovered as a little surprice and bonus for attendees.
```

Comment:
I think it's "we `did` not" instead of "we not"

Reply:
Corrected. Thanks for checking!